components/org.wso2.carbon.identity.api.server.certificate.validation.management/org.wso2.carbon.identity.api.server.certificate.validation.management.v1/src/main/java/org/wso2/carbon/identity/api/server/certificate/validation/management/v1/util/ModelMapperUtil.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.certificate.validation.management.v1.util;
+
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.CACertificate;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.CACertificates;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.Validator;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.Validators;
+import org.wso2.carbon.identity.x509Certificate.validation.model.CACertificateInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class for Certificate Validation Management endpoint.
```

Comment:
Is the description correct?

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/834/commits/a768a6bfccc225291c096836167adc0f03fda24c